### ISSUE TITLE
Component accessibility audit

### ISSUE THREAD

**Issue by ChrisKnightLDN** [Greater-London-Authority/ldn-viz-tools#367]
Working though components to check and address accessibility issues

**UI**
- [x] #736
- [x] #735
- [x] #737
- [x] #771
- [x] CheckboxGroup (tooltip violation)
- [x] CopyButton (button violation)
- [x] #763
- [x] #742
- [ ] #745
- [x] #772
- [x] Header
- [x] #764
- [x] #743
- [x] #748
- [x] #744
- [x] #773
- [ ] #749
- [x] #750
- [x] #741
- [x] NonIdealState
- [x] PlacardButton
- [x] PlaceholderImage
- [x] #751
- [x] #752
- [ ] #753
- [x] #746
- [x] #755
- [x] #757
- [x] #758
- [x] #759
- [x] ThemeSwitcher
- [x] #760
- [x] #738
- [x] UniformInput

**Apps**
- [x] #761

**Charts**
- [x] #762
- [x] ChartContainer?

**Maps**
- [ ] #765
- [x] #766
- [x] MapControlFullScreen
- [x] #767
- [x] #768
- [x] MapControlPan
- [x] MapControlRefresh
- [x] MapControlZoom
- [ ] #769
- [x] MapDeckOverlay
- [x] MapLayerSource
- [x] MapLayerView
- [ ] #770

**Tables**
- [x] Table

**Missing in UI**
- [ ] Footer
- [ ] Date Picker
- [ ] Sliders (range & single)

**Missing in Charts**
- [ ] Observable plot wrapper vs other? (need debate on scope & purpose of library, opinionated bar charts or only novel viz?)  

**Missing in Maps**
- [ ] Visual & non visual elements (/utils) (/maps/utils)
- [ ] Map layer handling UI, inc:
- [ ] Layer groups / sub groups
- [ ] Layer legends (based on types)
- [ ] Layer relationships (radio (either or) vs checkbox (many)) 
- [ ] Layer opacity
- [ ] Layer color picker
- [ ] Layer order dragging (scope?)

**Comment by ChrisKnightLDN** [comment]
closing as this work is now complete/ ongoing elsewhere
